architecture-decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
+
+Date: 2020-11-16
+
+## Status
+
+Accepted
+
+## Context
+
+We need to record the architectural decisions made on this project.
```

Comment:
should we add that we discussed it between the two of us? Either here or in the `Decision` section?

Reply:
yeah, i could add something. this was simply the result of what the tool generates by default, but a little more context would probably be worthwhile

Reply:
i added a bit more. happy to tweak further if you have thoughts on what else might be valuable to capture